src/components/FieldTypes.jsx

```
@@ -0,0 +1,240 @@
+        <span className="select-tag">{value}</span>
+      ) : (
+        <span className="select-placeholder">Select...</span>
+      )}
+    </div>
+  );
+};
+
+// Checkbox Field Component
+export const CheckboxField = ({ value, onChange, onSave, onCancel, isEditing }) => {
+  const handleToggle = () => {
+    const newValue = !Boolean(value);
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)